platforms/macosx/src/GFontOSX.h

```diff
@@ -41,15 +37,11 @@
 */		
 class_export GFontOSX : public VGFont		
 {
-#ifdef SMUFL
+    friend class GDeviceOSX;
```

Comment:
friend declarations are now forbidden (see https://github.com/grame-cncm/guidolib/wiki/Guidelines). I know there are some in the code, but they're there for historical reasons (and should be dropped asap).

Reply:
OK.